docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -143,15 +218,43 @@ $ odo dev
 Your application is now running on the cluster
  - Forwarding from 127.0.0.1:40001 -> 3000
 
-Watching for changes in the current directory /Users/user/syncthing/dev/k8s/devfiles/express
+Watching for changes in the current directory /Users/user/express
+Press Ctrl+c to exit `odo dev` and delete resources from the cluster
+```
+  </TabItem>
+
+  <TabItem value="dotnet" label=".NET">
+
+Let's run `odo dev` to start development on your .NET application:
+
+```console
+$ odo dev
+  __
+ /  \__     Developing using the my-dotnet60-app Devfile
+ \__/  \    Namespace: default
+ /  \__/    odo version: v3.0.0-alpha2
+ \__/
+
+↪ Deploying to the cluster in developer mode
+ ✓  Waiting for Kubernetes resources [3s]
+ ✓  Syncing files into the container [2s]
+ ✓  Building your application in container on cluster [5s]
+ ✓  Executing the application [1s]
+
+Your application is now running on the cluster
+ - Forwarding from 127.0.0.1:40001 -> 8080
+
+Watching for changes in the current directory /Users/user/dotnet
 Press Ctrl+c to exit `odo dev` and delete resources from the cluster
 ```
 
-You can now access the application at [127.0.0.1:40001](http://127.0.0.1:40001) in your local browser and start your development loop. `odo` will watch for changes and push the code for real-time updates.
 
   </TabItem>
 </Tabs>
 
+You can now access the application at [127.0.0.1:40001](http://127.0.0.1:40001) in your local browser and start your development loop. `odo` will watch for changes and push the code for real-time updates.
+
+
 ## Step 3. Deploying your application to the world (`odo deploy`)
```

Comment:
"to the world", really? We still don't claim odo to be a production ready tool. Inner loop and outer loop, that's it. Nitpicking, I know. Also, not a part of this PR, I know. 😉

Reply:
Want to open a PR to fix it 😜